<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebconfGui</class>
 <widget class="QWidget" name="DebconfGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>293</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="titleL">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string notr="true">Question Title</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="iconL">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">Icon</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPB">
       <property name="text">
        <string notr="true">Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="backPB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string notr="true">Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextPB">
       <property name="text">
        <string notr="true">Continue</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>436</width>
        <height>195</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nextPB</tabstop>
  <tabstop>backPB</tabstop>
  <tabstop>cancelPB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>nextPB</sender>
   <signal>clicked(bool)</signal>
   <receiver>nextPB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextPB</sender>
   <signal>clicked(bool)</signal>
   <receiver>backPB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backPB</sender>
   <signal>clicked(bool)</signal>
   <receiver>nextPB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backPB</sender>
   <signal>clicked(bool)</signal>
   <receiver>backPB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
